apis/vshn/v1/dbaas_vshn_keycloak.go

```diff
@@ -100,6 +101,19 @@ type VSHNKeycloakServiceSpec struct {
 	// PostgreSQLParameters can be used to set any supported setting in the
 	// underlying PostgreSQL instance.
 	PostgreSQLParameters *VSHNPostgreSQLParameters `json:"postgreSQLParameters,omitempty"`
+
+	// CustomThemeProviderImage can be used to provide an image with custom themes and providers.
+	// The themes need to be be placed in the `/themes` directory of the custom image.
+	// the providers need to be placed in the `/providers` directory of the custom image.
+	CustomThemeProvider VSHNKeycloakCustomThemeProvider `json:"customThemeProvider,omitempty"`
```

Comment:
Maybe another name for this? The comments imply that this thing handles providers and themes, naming is hard and the variable sounds like it only provides themes.

Reply:
Changed it to something more meaningful